doc/sphinx/proofs/writing-proofs/rewriting.rst

```diff
@@ -384,297 +381,248 @@ Rewriting with definitional equality
            exact H.
          Qed.
 
-.. _performingcomputations:
+.. _applyingconversionrules:
 
-Performing computations
----------------------------
+Applying conversion rules
+-------------------------
+
+These tactics apply reductions and expansions, replacing :term:`convertible` subterms
+with others that are equal by definition in |CiC|.
+They implement different specialized uses of the
+:tacn:`change` tactic.  Other ways to apply these reductions are through
+the :cmd:`Eval` command, the `Eval` clause in the :cmd:`Definition`/:cmd:`Example`
+command and the :tacn:`eval` tactic.
+
+Tactics described in this section include:
+
+- :tacn:`simpl` and :tacn:`cbn`, which are "clever" tactics meant to give the most
+  readable result
+- :tacn:`lazy` and :tacn:`cbv`, which allow precise selection of which reduction
+  rules to apply
+- :tacn:`vm_compute` and :tacn:`native_compute`, which are performance-oriented.
+
+Conversion tactics, with two exceptions, only change the types and contexts
+of existential variables
+and leave the proof term unchanged.  (The :tacn:`vm_compute` and :tacn:`native_compute`
+tactics change existential variables in a way similar to other conversions while
+also adding a single explicit cast to the proof term to tell the kernel
+which reduction engine to use.)  For example:
```

Comment:
This information is important and perhaps could be moved (or repeated) into the "vm_compute and native_compute" section. Also, some cross-links with the section which documents these casts would be welcome: https://coq.github.io/doc/master/refman/language/core/definitions.html#type-cast

Reply:
Added links from the comment about vm_compute and native_compute to the Type casts section.  And repeated the info at *_compute.